en/2-deeds.md

```diff
@@ -24,19 +24,19 @@ There are four types of Deeds, each providing different capabilities in terms of
 
 ## 2.3. Deeds ownership
 
-A Deed gives its owner the right to USE it, RENT it, or SELL it.
+A Deed gives its owner the right to USE, RENT, or SELL it.
 
-- USE – The owner can use the Deed to run a hub for his own organization.
+- USE – The owner can use the Deed to run a hub for his organization.
 - RENT - The owner can also rent the Deed to another organization and earn passive income.
 - SELL - The owner can sell it on a secondary NFT marketplace such as [OpenSea](https://opensea.io).
 
-Being an active holder or renter of a Deed, the organization is entitled to MEED rewards according to the “mintium” model described below (see 2.6).
+Being an active holder or renter of a Deed, the organization is entitled to MEED rewards according to the "Mintium" model described below (see 2.6).
 
 ## 2.4. Community Hubs
 
-In today's competitive market, organizations seek to attract and retain top talent not only through compensation but also by enhancing work engagement. The WoM addresses this challenge with a unique, tokenized reward system embodied in Community Hubs.
+Organizations seek to attract and retain top talent in today's competitive market through compensation and enhancing work engagement. The WoM addresses this challenge with a unique, tokenized reward system embodied in Community Hubs.
```

Comment:
```suggestion
Organizations seek to attract and retain top talent in today's competitive market through compensation and enhancing work engagement. The WoM addresses this challenge with a distinctive, tokenized reward system incorporated in Community Hubs.
```

Reply:
Using proper-sounding words that look professional and appealing gives this sentence a catchy look/feeling. Just like the words that were used in this sentence to replace the previous ones.